src/client/utils/browser.js

```diff
@@ -32,16 +26,46 @@ export var isIE11 = /trident\/7.0/.test(userAgent) && !(browser.name === 'msie'
 if (isIE11)
     majorBrowserVersion = 11;
```

Comment:
I think better use ternary in this case, something like this:
`var majorBrowserVersion = isIE11 ? parseInt(browser.version, 10) : 11;`
just swap strings